orion/handlers/grpc/grpc.go

```diff
@@ -31,10 +33,20 @@ type grpcHandler struct {
 
 func (g *grpcHandler) init() {
 	if g.grpcServer == nil {
-		g.grpcServer = grpc.NewServer(
-			grpc.UnaryInterceptor(g.grpcInterceptor()),
-			grpc.StreamInterceptor(g.grpcStreamInterceptor()),
-		)
+		var opts []grpc.ServerOption
+		if g.grpcInterceptor() != nil {
+			opts = append(opts, grpc.UnaryInterceptor(g.grpcInterceptor()))
+		}
+		if g.grpcStreamInterceptor() != nil {
+			opts = append(opts, grpc.StreamInterceptor(g.grpcStreamInterceptor()))
+		}
+		if g.config.CustomCodec != nil {
+			opts = append(opts, grpc.CustomCodec(*g.config.CustomCodec))
```

Comment:
`CustomCodec` is deprecated: https://pkg.go.dev/google.golang.org/grpc#CustomCodec